runtime/platform.c

```diff
@@ -225,15 +403,14 @@ void caml_mem_unmap(void* mem, uintnat size)
 #define Max_sleep_ns  1000000000 //  1 s
 
 unsigned caml_plat_spin_wait(unsigned spins,
```

Comment:
This function interface is potentially confusing. It doesn't spin, it just waits, and it does so for a number of nanoseconds, not a number of spins. Maybe it should be called caml_wait_for_ns or something. If the argument is called `spins` then maybe it should divide by a constant (which could be 1) called something like `SPINS_PER_NS` (or multiply by `NS_PER_SPIN`, I guess), with a comment explaining what's going on there. All just for clarity, of course - the compiled code will stay the same.